BoldGrid/boldgrid-seo
# Branch 1.6.11

Resolves #42 
Resolves #45 
Resolves #44

## Testing Instructions ##

1. Install the test RC: https://downloads.wordpress.org/plugin/boldgrid-easy-seo.1.6.11-rc1.zip
2. Ensure that the word count display updates when adding / removing words from a page or post.
3. Ensure that the indicators ( Red / Yellow / Green ) are showing Icons ( X / Exclamation / Check Mark ) instead of just colors.
4. Ensure that no fatal errors occur.
